### PR TITLE
Guest Inviter role is now supported

### PR DESCRIPTION
**Pull request by somakbhattacharyya**
Updating the documentation to reflect the new feature support.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/add-guests.md [hunk 1]
```diff
@@ -30,11 +30,10 @@ Anyone with a business or consumer email account, such as Outlook, Gmail, or oth
...
 
-> [!NOTE] 
-> This does not apply when **Admins and users in the guest inviter role can invite** is enabled. This is because the guest inviter role isn't supported in Teams.
-
 - Add guests to your organization through Azure Active Directory (Azure AD) B2B collaboration. Azure AD B2B collaboration allows a global admin to invite and authorize a set of external users by uploading a comma-separated values (CSV) file of no more than 2,000 lines to the B2B collaboration portal. For more details, check out [Azure Active Directory B2B collaboration](https://go.microsoft.com/fwlink/p/?linkid=826383).
 
+Admins can also delegate permissions to add guest to others in their organization by assigning the Guest Inviter role. For more information, see [Configure external sharing](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/delegate-invitations)
```
Comment (JohanFreelancer9) [inline Teams/add-guests.md:35]:
```suggestion
Admins can also delegate permissions to add guests to others in their organization by assigning the Guest Inviter role. For more information, see [Configure external sharing](https://docs.microsoft.com/azure/active-directory/external-identities/delegate-invitations).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**